README.md

```diff
@@ -60,6 +60,7 @@ with the latest released image sha.
 ```kotlin
 plugins {
   application
+  id("com.google.cloud.tools.jib") version "3.1.4"
```

Comment:
not a big deal, but let's set this to the latest for now, which is currently `3.3.1`